src/components/organisms/cards/cards.html

```diff
@@ -0,0 +1,62 @@
+<div class="row row-cols-1 row-cols-md-3">
+  <div class="col">
+    <div class="card">
+      <p class="title">CONCEPT</p>
+      <figure class="img-wrapper">
+        <img src="../../../img/card-1.jpg" class="card-img" />
+      </figure>
+      <div class="card-triangle"></div>
```

Comment:
今の状態だとカードの横幅が画面幅に合わせて滑らかに変化するので、任意の画面幅で3つのカードの長さを統一するのは結構難しいと思います。

そこで[container](https://getbootstrap.jp/docs/5.0/layout/containers/)を使ってみてください！containerはbreakpoint毎に固定幅を設定してくれます。
その後、paddingなどを追加して調整してみてください。

```suggestion
<div class="container">
<div class="row row-cols-1 row-cols-md-3">
  <div class="col">
    <div class="card">
      <p class="title">CONCEPT</p>
      <figure class="img-wrapper">
        <img src="../../../img/card-1.jpg" class="card-img" />
      </figure>
      <div class="card-triangle"></div>
```